ports/esp32s2/bindings/espidf/__init__.h

```diff
@@ -27,8 +27,16 @@
 #ifndef MICROPY_INCLUDED_ESP32S2_BINDINGS_ESPIDF___INIT___H
 #define MICROPY_INCLUDED_ESP32S2_BINDINGS_ESPIDF___INIT___H
 
+#include "esp_err.h"
+#include "py/mpconfig.h"
+#include "py/obj.h"
+
+extern const mp_obj_type_t mp_type_espidf_IDFError;
 extern const mp_obj_type_t mp_type_espidf_MemoryError;
 
 NORETURN void mp_raise_espidf_MemoryError(void);
 
+void raise_esp_error(esp_err_t err) NORETURN;
+#define ESP_CALL_RAISE(x) do { int res = (x); if(res != ESP_OK) raise_esp_error(res); } while(0)
```

Comment:
I think this is clearer. In particular, I don't think the macro name needs "call" in the name.

```suggestion
#define CHECK_ESP_RESULT(x) do { int res = (x); if(res != ESP_OK) raise_esp_error(res); } while(0)
```

Reply:
Thanks, done